csrc/buffer.h

```diff
@@ -402,6 +402,7 @@ class jni_borrow {
     }
 
     size_t len() const { return m_length; }
+    size_t size() const { return len(); }
```

Comment:
nit: I'd avoid adding aliases like this unless there's a compatibility reason. Otherwise, we end up proliferating different yet identical methods in different parts of the code base for no reason.

Reply:
Oops, I agree, I meant to remove it before submitting the PR.